docs/_layouts/pages.html

```diff
@@ -70,3 +151,54 @@
     {{ content }}
   </div>
 </div>
+
+<script>
+
+  var filterInput = document.getElementById('filter-input');
+  var headings = document.getElementsByClassName('heading');
+  var tocElements = document.getElementsByClassName('example-names');
+  
+
+  filterInput.addEventListener('keyup', function (e) {
+    var value = this.value.toLowerCase();
+    var element;
+
+    for (i=0; i < headings.length; i++) {
+      if (!value || value == undefined || value == "" || value.length == 0) {
+        headings[i].classList.remove('display-none');
+      } else { 
+        headings[i].classList.add('display-none');
+      }
+    }
+
+
+ 
```

Comment:
We don't use the double (or triple) newline formatting pattern in the GL JS codebase.